docs/content/misc/tasks.md

```diff
@@ -251,7 +251,6 @@ The indexing service can also run real-time tasks. These tasks effectively trans
       "maxRowsInMemory": 500000,
       "intermediatePersistPeriod": "PT10m",
       "windowPeriod": "PT10m",
-      "basePersistDirectory": "\/tmp\/realtime\/basePersist",
```

Comment:
Eric says it's a feature that basePersistDirectory is ignored for realtime tasks (as opposed to realtime nodes).